pkg/selinux/selinux.go

```diff
@@ -269,20 +268,37 @@ func SelinuxGetEnforceMode() int {
 	return Disabled
 }
 
+func mcsPath(mcs string) string {
+	filename := fmt.Sprintf("/run/rkt/%s", mcs)
+	return filename
+}
+
 func mcsAdd(mcs string) error {
-	if mcsList[mcs] {
-		return fmt.Errorf("MCS Label already exists")
+	filename := mcsPath(mcs)
+	file, err := os.OpenFile(filename, os.O_CREATE|os.O_EXCL|os.O_RDONLY, 0644)
+	if err != nil {
+		if os.IsExist(err) {
+			return fmt.Errorf("MCS label already exists")
+		} else {
+			return fmt.Errorf("unable to test MCS: %s", err)
+		}
 	}
-	mcsList[mcs] = true
+	file.Close()
 	return nil
 }
 
 func mcsDelete(mcs string) {
-	mcsList[mcs] = false
+	filename := mcsPath(mcs)
+	os.Remove(filename)
```

Comment:
I am not so familiar with SELinux, but I guess the mcs string cannot be controlled by the user and mcs cannot contain "../../etc/passwd", right? It seems fine after a quick look, but I prefer to ask :)